src/nav/infrastructure.yml

```diff
@@ -142,6 +142,8 @@ pages:
             path: /docs/infrastructure/host-integrations/host-integrations-list/active-directory-integration
           - title: Aerospike integration
             path: /docs/infrastructure/host-integrations/host-integrations-list/aerospike-integration
+          - title: Monitor Apache Airflow with OpenTelemetry 
```

Comment:
Seems odd the way this is alphabetical order. Can we lose the "Monitor" here of shift it down the Ms?

Reply:
@paperclypse I removed "Monitor."